portfolio/src/main/webapp/style.css

```diff
@@ -106,6 +114,31 @@ td p {
   color: rgb(61,95,121);
 }
 
+#hobbies-div {
+  color: white;
+}
+
+.hobbies-page-a {
+  color: rgb(250,220,220);
+  text-decoration: none;
+}
+
+.hobbies-description {
+  padding: 3rem 4rem 3rem 1rem;
+}
+
+.hobbies-imgs tr td img {
+  height: 8rem;
+}
+
+.hobbies-image {
```

Comment:
I don't see where it is used?

Reply:
Ah I must have left that from a previous version, will remove.